main_app/migrations/0006_auto_20210113_1531.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.4 on 2021-01-13 15:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main_app', '0005_park_rating'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='park',
+            name='email',
+            field=models.CharField(default='NO EMAIL', max_length=100),
+        ),
+        migrations.AddField(
+            model_name='park',
+            name='lat',
+            field=models.FloatField(default=0.0, max_length=20),
+        ),
+        migrations.AddField(
+            model_name='park',
+            name='lng',
+            field=models.FloatField(default=0.0, max_length=20),
+        ),
+    ]
```

Comment:
could you tell me the command you used to create the migrations? (Is there a command to add columns like there is for Rails: `rails g migration AddArbitraryToTables arbitrary:references`

Reply:
Yeah and I added it to the `README.md`, but the commands are (after you make changes to the model):

`python3 manage.py makemigrations`
`python3 manage.py migrate`